
sender:
  ros__parameters:
    pub_frequency: 1.0
    ping_pub_topic: "ping_msg"
    ping_sub_topic: "pong_msg"
    
    msg_num:  30
  
    qos_depth:  1000
    qos_best_effort: True
    qos_durability_transient_local: False
    qos_history_keep_all: False
